src/Application/FileProcessor.php

```diff
@@ -59,6 +70,8 @@ public function processFile(File $file, Configuration $configuration): FileProce
             return new FileProcessResult([$parsingSystemError], null);
         }
 
+        $this->cacheFileDependencies($file, $allFiles);
```

Comment:
After parsing the file, we calculate the dependencies for this file and cache them